### DISCUSSION TITLE
Why are the two different settings for AutoDJ?

### DISCUSSION THREAD

**Discussion by DicipleCrimson** [AzuraCast/AzuraCast#6972] [Q&A]
I am using AzuraCast with live DJs only. So I have no use for any AutoDJ function for any of the users. What's a little confusing to me is that there seem to be two different settings for AutoDJ. One tab on the station profile. But also another tab on the mount point for each station. Do they have different functions?

Why I am asking is really because I can't find a setting for bitrate on the outgoing stream anywhere. Except if I turn on AutoDJ for the mount point. Then there are options to change format (mp3, ogg etc) and bitrate. But since we aren't using AutoDJ I left it disabled on all stations. My DJs all use mp3 in various bitrates in their streaming software. And it seems to be working fine. Does that mean the mount point accepts any format and any bitrate from DJ streaming software if AutoDJ is disabled?

**Comment by gAlleb** [comment]
Hi @DicipleCrimson !

> One tab on the station profile. But also another tab on the mount point for each station. Do they have different functions?

1. Tab on the station profile allows you configure different options of AutoDJ (Liquidsoap).
2.  Tab on the mount point for each station allows you to configure the bitrate of the stream which is being generated by AutoDJ (Liquidsoap).

> I can't find a setting for bitrate on the outgoing stream anywhere.

If you don't use AutoDJ - you can't set the bitrate because no stream is generated by AutoDJ. Since your DJs set bitrates in their streaming software and generate their stream from their software - you don't have to set any bitrate in Azuracast for that. Indeed mount point will accept any format and any bitrate from DJ and their streaming software.

> Does that mean the mount point accepts any format and any bitrate from DJ streaming software if AutoDJ is disabled?

Right! (As long as Icecast server supports it).

All is good)
